SUPPORTED_LANGUAGES.md

```diff
@@ -90,6 +90,7 @@ Languages that listed a **Package** below are 3rd party languages and are not bu
 | Handlebars              | handlebars, hbs, html.hbs, html.handlebars        | |
 | Haskell                 | haskell, hs            |         |
 | Haxe                    | haxe, hx               |         |
+| High-level shader language| hlsl                | [highlightjs-hlsl](https://github.com/stef-levesque/highlightjs-hlsl) |
```

Comment:
Is it more commonly recognized as HLSL to most? We're not afraid of using abbreviations in this column if they are common.

Reply:
Both are used, I'm following the same convention as `OpenGL Shading Language | glsl`

Reply:
OK, we're not 100% consistent here anyways prolly. :-)  Resolving.